mainwindow.h.autosave

```diff
@@ -0,0 +1,48 @@
+#ifndef MAINWINDOW_H
```

Comment:
probably don't want to check this file in and want to add *.autosave to .gitignore